### PR TITLE
Create harbour-scintillon-zh_CN.ts

### PR DESCRIPTION
**Pull request by dashinfantry**

### CODE REVIEW

File: apps/sailfish/translations/harbour-scintillon-zh_CN.ts [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+    </message>
+    <message id="scintillon-alarm_placeholder_no_hub">
+        <location filename="../qml/pages/AlarmsPage.qml" line="27"/>
+        <source>Not connected to a Hue Hub</source>
+        <translation>没有连接到 Hue Hub</translation>
+    </message>
+    <message id="scintillon-alarm_list_title">
+        <location filename="../qml/pages/AlarmsPage.qml" line="33"/>
+        <source>Alarms and timers</source>
+        <translation><通知及计时器/translation>
```
Comment (llewelld) [inline apps/sailfish/translations/harbour-scintillon-zh_CN.ts]: The opening bracket of the closing tag is just out of place here:
```
<translation><通知及计时器/translation>
```
should be:
```
<translation>通知及计时器</translation>
```
If you could please fix this, then it looks good to go.

**Review by llewelld — CHANGES_REQUESTED** [review]
Thanks, this is great. Just one minor fix needed.

**Review by dashinfantry — COMMENTED** [review]
changed now

**Comment by llewelld** [comment]
Thanks, but I'm seeing some strange artefacts after this latest change (e.g. an [extra DOCTYPE declaration on line 20](https://github.com/llewelld/shine/pull/1/files/db8bf6db47a06ae4bfff9afa56f396620b78b8a7#diff-efeffcc1dc05859ba44adf862fac4098R20-R22)). Is it possible something got accidentally rearranged in your latest commit?

**Comment by dashinfantry** [comment]
What? 星期一, 26 八月 2019, 00:26上午 +08:00 发件人 David Llewellyn-Jones  notifications@github.com :

**Review by llewelld — APPROVED** [review]

**Pull request merged**